t/t1002-sign-munge.t

```diff
@@ -96,12 +106,17 @@ test_expect_success 'message with altered munge cred fails verify' '
 '
 
 # N.B. max-ttl = (exactly) -100 is allowed for testing
-test_expect_success 'create sign.toml with max-ttl=-100' '
+test_expect_success 'recreate sign.toml with max-ttl=-100' '
 	cat >sign.toml <<-EOT
 	[sign]
 	max-ttl = -100
 	default-type = "munge"
 	allowed-types = [ "munge" ]
+	EOT
+'
+
+test_expect_success SIDEMUNGE 'adding munge.socket-path to sign.toml' '
+	cat >>sign.toml <<-EOT
 	[sign.munge]
```

Comment:
Instead of skipping configuration test here, could we add the system munge socket path explicitly?
Could this allow one of the redundant `'adding munge.socket-path'` "tests" to go away?

Reply:
Meant to add that system munge socket-path is available via `munged --help | grep socket= | sed 's/.*\[\(.*\)\]$/\1/'
`

Reply:
That was my first thought too, but I'm not sure how to reliably determine the path?

Reply:
I think if `munge | unmunge` works with setting `--socket`, then it can be assumed that the MUNGE daemon is using the default socket-path, which is printed in `--help` output (see above)

Reply:
Got it, thanks!